AEPMessaging/Sources/LiveActivity/LiveActivityPushToStartTokenStore.swift

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
I guess this file is only removed from the xcode project and from not the folder..
Can we delete `LiveActivityPushToStartTokenStore` and `LiveActivityUpdateTokenStore` files ?